android/src/main/java/nl/xguard/flic2/callback/flic2ButtonCallback.java

```diff
@@ -26,34 +26,39 @@ public void onDisconnect(Flic2Button button) {
         Log.d(TAG, "onDisconnect() called with: button = [" + button + "]");
         mReactEvent.send(button, ReactEvent.EVENT_BUTTON_STATUS_DISCONNECTED);
     }
+    
     @Override
     public void onConnect(Flic2Button button) {
         Log.d(TAG, "onConnect() called with: button = [" + button + "]");
         mReactEvent.send(button, ReactEvent.EVENT_BUTTON_STATUS_CONNECTION_COMPLETED);
     }
+
     @Override
     public void onFailure(Flic2Button button, int errorCode, int subCode) {
-        Log.d(TAG, "onFailure() called with: button = [" + button + "]");
+        Log.d(TAG, "onFailure() called with: button = [" + button + "]" + " error: " + errorCode);
```

Comment:
the error code is logge dhere but not return (and the subCode), why not pass those to JS?